server/src/storage/segment.rs

```diff
@@ -142,7 +142,7 @@ pub struct SpilledSegment {
 impl SpilledSegment {
     /// Spill the data in `batch` to a Parquet file, and return a spilled segment with the path.
     pub fn new(key: String, batch: RecordBatch) -> Self {
-        let folder_path = format!("uncompressed/{}", key);
+        let folder_path = format!("storage/{}/uncompressed", key);
```

Comment:
Is "storage" a placeholder until the `StorageEngine` is integrated and the user-defined data folder can be used?

Reply:
Yes, I have added a TODO to change this after a user-defined data folder is given.